x/programs/rust/examples/token/Cargo.toml

```diff
@@ -11,3 +11,12 @@ wasmlanche_sdk = { version = "0.1.0", path = "../../wasmlanche_sdk" }
 
 [lib]
 crate-type = ["cdylib"] # set the crate(needed for cargo build to work properly)
+
+# when creating a new project, you can uncomment the following lines to enable optimizations
+# [profile.release]
+# opt-level = 3
+# lto = true
+# codegen-units = 1
+# overflow-checks = true
+# # https://doc.rust-lang.org/book/ch09-01-unrecoverable-errors-with-panic.html#unwinding-the-stack-or-aborting-in-response-to-a-panic
+# panic = 'abort'
```

Comment:
is this considered an optimization or required?

Reply:
panic=abort is the default for the `wasm32-unknown-unknown` target which is now our default target. So this is just visibility to the default in case we wanted to changed it. I do have a question of how well the memory is cleaned up in this case by the OS.

Reply:
The rust book describes this as well as says "dont panic" :)
```
Ultimately, panics translate into aborts in wasm32-unknown-unknown anyways, so this gives you the same behavior but without the code bloat.
```
ref. https://rustwasm.github.io/docs/book/reference/code-size.html?highlight=panic#avoid-panicking